PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/function/runner/core/FunctionUtils.java

```diff
@@ -208,7 +208,7 @@ public static boolean isFunctionClassAnnotated(final PsiMethod method) {
 
     @AzureOperation(
         name = "function.copy_settings",
-        params = {"$model.getName"},
+        params = {"localSettingJson", "stagingFolder"},
```

Comment:
localSettingJson is a path and may have user's information, this path shall not be recorded in our telemtry

Reply:
parameters will not be recorded in telemetry.